www/src/components/CodeEditor/CodeEditor.module.scss

```diff
@@ -45,6 +61,10 @@
   @include breakpoint("small-") {
     border-radius: 10px 10px 0 0 !important;
   }
+
+  :global pre.prism-code[contenteditable] {
+    height: 280px !important;
```

Comment:
I've added a `max-height` to the live code examples. I feel it's obvious to scroll, and is visually balanced.